communicator/ssh/provisioner.go

```diff
@@ -254,6 +269,31 @@ func buildSSHClientConfig(opts sshClientConfigOpts) (*ssh.ClientConfig, error) {
 	return conf, nil
 }
 
+// Create a Cert Signer and return ssh.AuthMethod
+func signCertWithPrivateKey(pk string, certificate string) (ssh.AuthMethod, error) {
```

Comment:
Is this something we can add unit tests for? My knowledge of SSH and certificates is fairly shallow, so forgive me if I'm asking a silly question

Reply:
I don’t know how the unit tests work for this provisioner.